data/pathfinder/paizo/roleplaying_game/advanced_class_guide/acg_abilities_class.lst

```diff
@@ -2482,60 +2482,62 @@ Vengeance Strike						KEY:Wild Child ~ Vengeance Strike								CATEGORY:Special
+
+Weapon and Armor Proficiency			KEY:Ecclesitheurge ~ Weapon and Armor Proficiency				CATEGORY:Special Ability	TYPE:Ecclesitheurge Class Feature.Cleric Class Feature.SpecialQuality.ClericWeaponProficiencies.ClericArmorProficiencies					DESC:An ecclesitheurge is proficient with the club, dagger, heavy crossbow, light crossbow, and quarterstaff, but he's not proficient with any type of armor or shield. This replaces the cleric's weapon and armor proficiencies.					AUTO:WEAPONPROF|Club|Dagger|Crossbow (Heavy)|Crossbow (Light)|Quarterstaff					SOURCEPAGE:p.91
+Ecclesitheurge's Vow					KEY:Ecclesitheurge ~ Ecclesitheurge's Vow						CATEGORY:Special Ability	TYPE:Ecclesitheurge Class Feature.Cleric Class Feature.SpecialQuality																		DESC:An ecclesitheurge makes a vow to his deity to be protected solely by his faith, not by armor or shields. An ecclesitheurge who wears armor or uses a shield is unable to use his blessing of the faithful ability, use cleric domain powers, or cast cleric spells.				SOURCEPAGE:p.91
+Blessing of the Faithful				KEY:Ecclesitheurge ~ Blessing of the Faithful					CATEGORY:Special Ability	TYPE:Ecclesitheurge Class Feature.Cleric Class Feature.SpecialQuality.Supernatural															DESC:As a standard action, the ecclesitheurge can bless one ally within close range (%1 ft.). A blessed ally gains a +2 sacred or profane bonus (depending on whether the ecclesitheurge channels positive or negative energy) on attack rolls, skill checks, ability checks, or saving throws or to AC until the ecclesitheurge's next turn. The ecclesitheurge can expend 1 use of channel energy when activating this ability to increase the duration to a number of rounds equal to the number of dice of his channel energy.|25+(floor(ClericLVL/2)*5)																																																																																																																																																																																																																																																																																																																																									SOURCEPAGE:p.91
+####:COMMENT:Domain Mastery uses standard spell slots to prepare additional domain spells. Unable to implement.
```

Comment:
Basically we add a spell slot to each level of 1 through 9? Easily done, and leave it to the player to enforce.